src/repositories/solRepository.ts

```diff
@@ -0,0 +1,22 @@
+import { MysqlDataSource } from '../config/database';
+import { Sol } from '../entity/Sol';
+
+export class SolRepository {
+  /**
+   *Saves each newSol in the soles array is saved to the database, in case it's not already there
+   * @param soles Array of "soles" from: SolController and NasaService
+   * @returns {Promise<void>} It just saves data to the soles table
```

Comment:
não precisa documentar o returns quando a função for do tipo void, pois já se entende que não há retorno esperado